metadata.json

```
@@ -1,12 +1,12 @@
 {
-  "name": "ccooke-collections",
-  "version": "0.1.2",
-  "author": "charlesc",
+  "name": "puppet-collections",
+  "version": "1.0.0",
+  "author": "ccooke",
```

We usually update this to Vox Pupuli, I'm not sure if the schema allows an array, need to be tested:
```suggestion
  "author": "Vox Pupuli",
```

```suggestion
  "author": ["ccooke", "Vox Pupuli"],
```

---

Strings only, it seems. Set to Vos Pupuli.